src/D2L.CodeStyle.Analyzers/Async/BlockingAnalyzer.cs

```diff
@@ -308,10 +314,15 @@ ICollection<IMethodSymbol> haveCallsToSomethingBlocking
 				}
 			);
 
-			if( earliestAncestor.IsKind( SyntaxKind.MethodDeclaration )
-			 || earliestAncestor.IsKind( SyntaxKind.LocalFunctionStatement )
-			) {
+			if( earliestAncestor.IsKind( SyntaxKind.MethodDeclaration ) ) {

```

Comment:
`earlieastAncestor is MathodDeclarationSyntax methodDecl`?

Reply:
I dno, I guess I did it because IsKind is cheaper but I don't really care.
I'm certainly not consistent about this.